.github/workflows/auto_translate.yml

```diff
@@ -0,0 +1,55 @@
+name: Auto translate
+on:
+  workflow_call:
+    inputs:
+      branch:
+        description: 'The branch to checkout and run the action in'
+        type: string
+        default: ${{ github.ref }}
+      action_branch:
+        description: 'The shared action branch to checkout'
+        type: string
+        default: main
+      python_version:
+        description: 'The python version to use'
+        type: string
+        default: "3.8"
+      locale_folder:
+        description: 'The base folder to look for localization files'
+        type: string
+        default: locale
+      
+
+jobs:
+  autotranslate:
+    env:
+      API_KEY: ${{secrets.DL_API_KEY}}
```

Comment:
What is this API key? Is it something that should be available as an org secret?

Reply:
I believe this is [DeepL](https://www.deepl.com/pro-api?cta=header-pro-api). I'm not sure if usage will fit into the free tier with more than a few active repos so the org might need a paid key to implement this everywhere

Reply:
free tier (500k) api key. not quite clear about the irritation.